src/kernel-install/kernel-install.c

```diff
@@ -1086,32 +1087,65 @@ static int verb_inspect(int argc, char *argv[], void *userdata) {
 
         c->action = ACTION_INSPECT;
 
+        /* Unfortunately because of backwards compatibility the argument order is reversed here compared to
+         * "add". */
```

Comment:
why? i mean, you could allow `kernel-install inspect [[VERSION] IMAGE]`

i.e. if one arg is specified it's the image, if two are specified it's the version first, and the image 2nd?

Reply:
Well the problem is the current condition is `if (argc >= 2) {` so technically we'd still be breaking compat if we applied your suggestion.